static/css/authorization-quiz.css

```
@@ -0,0 +1,535 @@
+  --quiz-navy-light: #0b4a74;
+  --quiz-blue: #1eaef7;
+  --quiz-action: #0062ba;
+  --quiz-ink: #172536;
+  --quiz-muted: #526173;
+  --quiz-line: #c8d3dc;
+  --quiz-soft: #eef5fa;
+  --quiz-success: #0b6b3a;
+  --quiz-success-bg: #e8f6ee;
+  --quiz-error: #a1261d;
+  --quiz-error-bg: #fff0ef;
+  color: var(--quiz-ink);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)